### DISCUSSION TITLE
Request feedback on BERTopic v. LDA article.

### DISCUSSION THREAD

**Discussion by drob-xx** [MaartenGr/BERTopic#597] [Show and tell]
Hi All,

I've drafted a Medium article that I'll be publishing with [Towards Data Science](https://towardsdatascience.com/). The article is a comparison between LDA and BERTopic that focusses on work I've been doing around HDBSCAN tuning. I am soliciting constructive criticism before publishing the article. I'm particularly interested in feedback from those experienced with LDA and/or HDBSCAN tuning. 

Here's the link to a draft version of: [Is LDA Topic Modeling Dead?](https://medium.com/@drob707/is-lda-topic-modeling-dead-9543c18488fa). 

I also prepared a companion Tableau presentation [here](https://public.tableau.com/views/IsLDATopicModelingDead/EmbeddingsStory?:language=en-US&publish=yes&:display_count=n&:origin=viz_share_link).

Feedback appreciated.

**Comment by MaartenGr** [comment]
Apologies for the late reply! It was a very interesting read and definitely nice to see such an in-depth comparison between LDA and BERTopic. Especially the Tableau dashboard for going through the interaction definitely helped 😄
